src/goto-programs/show_goto_functions_json.cpp

```diff
@@ -55,23 +55,26 @@ void show_goto_functions_jsont::show_goto_functions(
   const goto_functionst &goto_functions)
 {
   json_arrayt json_functions;
-  forall_goto_functions(it, goto_functions)
+  for(const auto &function_entry : goto_functions.function_map)
```

Comment:
As suggested squashed the commit (and fixed the typo) as as you said, was "introduced" by this PR.